php/Terminus/Collections/Instruments.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Terminus\Collections;
+use Terminus\Request;
+use Terminus\User;
+use Terminus\Instrument;
+use \Terminus_Command;
+
+class Instruments {
+  private $user;
+  private $models = array();
+
+  public function __construct($options = array()) {
+    $this->user = $options['user'];
+  }
+
+  public function fetch() {
+    $results = Terminus_Command::request(
+      'users',
+      $this->user->get('id'),
+      'instruments',
+      'GET'
+    );
+
+    foreach(get_object_vars($results['data']) as $id => $instrument_data) {
+      $instrument_data->id = $id;
+      $options             = array('user' => $this->user);
+      $this->models[$id]   = new Instrument(
+        $instrument_data,
+        $options
+      );
```

Comment:
I would prefer we didn't break arguments over multiple lines. If we're writing good method signatures, we should at most have 1 or 2 positional args and an options array().